cluster_tools/cluster_tools/pickling.py

```diff
@@ -45,27 +36,23 @@ def get_suitable_pickle_protocol():
 
 @warn_after("pickle.dumps", WARNING_TIMEOUT)
 def dumps(*args, **kwargs):
-    return pickle_strategy.dumps(
-        *args, protocol=get_suitable_pickle_protocol(), **kwargs
-    )
+    return pickle.dumps(*args, protocol=get_suitable_pickle_protocol(), **kwargs)
 
 
 @warn_after("pickle.dump", WARNING_TIMEOUT)
 def dump(*args, **kwargs):
-    return pickle_strategy.dump(
-        *args, protocol=get_suitable_pickle_protocol(), **kwargs
-    )
+    return pickle.dump(*args, protocol=get_suitable_pickle_protocol(), **kwargs)
 
 
 @warn_after("pickle.loads", WARNING_TIMEOUT)
 def loads(*args, **kwargs):
     assert (
         "custom_main_path" not in kwargs
     ), "loads does not implement support for the argument custom_main_path"
-    return pickle_strategy.loads(*args, **kwargs)
+    return pickle.loads(*args, **kwargs)
 
 
-class RenameUnpickler(pickle_strategy.Unpickler):
+class RenameUnpickler(pickle.Unpickler):
```

Comment:
Since cloudpickle does not expose `cloudpickle.Unpickler`, the support for cloudpickle was broken anyways (probably since 3 years, see [this PR](https://github.com/scalableminds/cluster_tools/pull/68/files#diff-263c407a6e6c37c0e1033d2c2022c66c6036de62eb52480b549059327822e794R58). I simply removed it now.

Reply:
Yes, that's probably a good call. Since cloudpickle caused some trouble back then, we switched back to the default pickling and haven't really had any need to switch back. So, let's remove it altogether :+1: